pkg/components/flatcar-linux-update-operator/component.go

```diff
@@ -26,10 +26,10 @@ import (
 	"github.com/kinvolk/lokomotive/pkg/k8sutil"
 )
 
-const componentName = "flatcar-linux-update-operator"
```

Comment:
This seems to be a minor cleanup, which should be committed separately.

Reply:
Done.